controllers/events.js

```diff
@@ -0,0 +1,75 @@
+const { NotFound } = require('../services/errors');
+
+class EventController {
+  constructor(eventsRepository) {
+    this.eventsRepository = eventsRepository;
+  }
+
+  getList = async (req, res, next) => {
+    try {
+      const events = await this.eventsRepository.findAllEvents(req.query);
+
+      return res.json({ events });
+    } catch (error) {
+      next(error);
+    }
+  };
+
+  getById = async (req, res, next) => {
+    const id = +req.params.eventId;
```

Comment:
this not the best validation of numbers. You can get `NaN` here and so unexpected errors.